app/services/app_store_list_service.rb

```diff
@@ -0,0 +1,29 @@
+module AppStoreListService
```

Comment:
To semi-future proof the whole "one single LAA data store" requirement, should this be named/structured as such?

Reply:
Yeah, I'm not sure about the best way to do that. My thinking is that as far as the provider app is concerned, there is a store of applications, that it can call "the app store", and it doesn't need to know or care whether "the" app store is a dedicated thing that's only used for CRM4 and 7, or whether it's an LAA-wide thing that also contains a bunch of other stuff. It's just the store, and it can be used to ask for specific lists of applications by this service that knows about the API of the relevant endpoint.

Reply:
Yeah that's true, I suppose there wouldn't be much to do and that's probably so far down the road it doesn't warrant worrying